src/var/distances.jl

```diff
@@ -130,6 +134,92 @@ function distance{T<:MutationType,A<:NucleotideAlphabet}(::Type{Count{T}}, seqs:
 end
 
 """
+    distance{T<:MutationType,A<:NucleotideAlphabet}(::Type{Count{T}}, seqs::Vector{BioSequence{A}}, width::Int, step::Int)
+
+A distance method which computes pairwise distances using a sliding window.
```

Comment:
The first summary line of a docstring should be an imperative form: "Compute pairwise distances using sliding window."